internal/app/app.go

```diff
@@ -55,7 +58,7 @@ func NewLogger() log.Logger {
 
 func NewClient(ctx context.Context, logger log.Logger) bot.Client {
 	configuredClient := config.Env.Client
-	logger.Info(ctx, fmt.Sprintf(
+	logger.Debug(ctx, fmt.Sprintf(
```

Comment:
I would actually keep this one as Info, since it is just a one-line log, runs only once, in the beginning, and brings a relevant information (the configured client)